frontend/components/community/navigation.tsx

```diff
@@ -167,7 +167,7 @@ export default function NavigationDisplay({
                      fontSize={`${fontSize.md}`}
                      fontWeight={`${fontWeight.normal}`}
                   >
-                     {'Join the Community'}
+                     {'Community'}
```

Comment:
This code refers to a page that has never been published:

![Screenshot 2023-12-14 at 08 39 02](https://github.com/iFixit/react-commerce/assets/4640135/305b520b-451b-47b7-80e7-943b0539b71a)

`Join the Community` in the navbar is sourced from Strapi, so no code changes are needed for that

Reply:
Yeah, there's a mirror on the PHP side [here](https://github.com/iFixit/ifixit/pull/51128)

I'm interested in narrowing the UI differences between the repos. This was a "value day" project to get that started.

Re: Community page - Personally, I prefer to change wholesale so there is less to worry about if we do use in the future. If we aren't ever going to use it, we should consider removing it entirely.